coffee-client/src/containers/coffee-detail.js

```diff
@@ -144,7 +141,11 @@ class CoffeeDetail extends Component {
 
     let reviewsarray = [reviews, "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasiem ullam corporis suscipit laboriosam, nisi ut aliquid ex consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?", "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasiem ullam corporis suscipit laboriosam, nisi ut aliquid ex consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur ?"];
 
-
+     let renderReviews = () => {
+       return reviewsarray.map(review => {
+         return <Paragraph key={uuid.v4()} top='2%'> " {review} " </Paragraph>;
```

Comment:
Using a random id for this is really ineffective, I prefer you to use the position of the element in the array to this.